linux_os/guide/system/apparmor/all_apparmor_profiles_in_enforce_complain_mode/ansible/shared.yml

```diff
@@ -0,0 +1,27 @@
+# platform = multi_platform_sle
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+{{{ ansible_instantiate_variables("var_apparmor_mode") }}}
+
+- name: Ensure all AppArmor Profiles are reloaded
+  ansible.builtin.shell: apparmor_parser -q -r /etc/apparmor.d/
```

Comment:
Can this and other tasks be executed with the `command` module instead of `shell` module?
The `command` module should be used unless `shell` module is explicitly required.